### ISSUE TITLE
Training a model from result of a previous model. Possible?

### ISSUE THREAD

**Issue by uq1** [BenAAndrew/Voice-Cloning-App#106]
Not an issue, more just a hypothetical question.

If I trained a model from a small dataset, using a pretrained model as a base, the results tend to turn out partially usable.

Some sentences work fine, others not so much. I am assuming most of the working ones are due to the pretrained model used, combined with some similarity in the training data compared to the pretrained model.

Now, if I then went ahead and generated a bunch of small sentences from the resulting model, manually removing the sentences with issues, then joining the good sentences back together into a single, long, audio file (and matching text file), what would be the result of training a new model from this new text?

Naturally, the resulting model would diverge more from the original voice, and also some loss of audio quality, but could this be used to make a small dataset into a longer one, still somewhat similar to the original speaker?

I am not  sure if anyone has tried this, but I think it may be worth looking into? If there is some sort of success, then maybe this is something that might help in the software itself (automation of this). It is hard to find large samples of voices with accurate transcriptions, maybe this could be a way around that?

**Comment by BenAAndrew** [comment]
Interesting question. To be honest, I have no idea. I suspect that feeding in synthesised clips as training data would probably not aid the model as the noise contained in these clips might reduce quality but I'm not sure so it's worth an experiment if you fancy it

**Comment by BenAAndrew** [comment]
Closing due to inactivity
